sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
   [PR 9233](https://github.com/Azure/azure-sdk-for-js/pull/9233)
   [PR 9284](https://github.com/Azure/azure-sdk-for-js/pull/9284)
 
+### Breaking Changes
+
+- `top` attribute of `ListRequestOptions` interface has been renamed to `maxCount`.
+  `options` argument of type `ListRequestOptions` is expected to be passed for the methods that deal with retrieving information of multiple entities, example - `getQueues`, `getQueuesRuntimeInfo`.
+  [PR 9664](https://github.com/Azure/azure-sdk-for-js/pull/9664)
```

Comment:
Typically, users will not be very familiar with the interface names especially if it is for input types, because when they write the code, they almost never have to even know about them.

```suggestion
- The property `top` in the options passed to any of the methods that get information for multiple entities like `getQueues` or `getQueuesRuntimeInfo` is renamed to `maxCount` [PR 9664](https://github.com/Azure/azure-sdk-for-js/pull/9664)
```